docs/integrations/integration-basics/supported-transformations.md

```diff
@@ -0,0 +1,125 @@
+
+Stream QL is documented in [Stream Worker Queries](../../cep/query-guide/), but only portions of that section apply to transformations.
+
+This page explains the difference between transformations and stream workers and lists the portions of Stream QL (the language used in stream worker queries) you can and cannot use in transformations.
+
+## Transformations vs. Stream Workers
+
+You can think of transformations as lightweight stream workers. You can use either one depending on your data pipeline and processing needs.
+
+### Input and Output
+
+Transformations have input and output defined automatically, and you cannot change them.
+
+You can define multiple inputs and outputs for stream workers, but you must set them up yourself.
+
+### Queries
+
+Transformations allow you to write one query, while stream workers allow you to write several queries. For more information about stream worker queries, refer to [Stream Worker Queries](../../cep/query-guide/).
+
+### Workflow
+
+Transformations only process, or transform, content going into or out of Macrometa collections via a connection.
+
+Stream workers process data streams, which can include Macrometa collections, but also support a variety of different [sources](../../cep/source/) and [sinks](../../cep/sink/).
+
+## Supported Stream QL
+
+Stream QL is documented in [Stream Worker Queries](../../cep/query-guide/), but only the following portions of that documentation apply to transformations. If your data pipeline needs are met by the following functionality, then a transformation might be the best solution:
+
+- [Query](../../cep/query-guide/query), except for the joins and stream worker examples.
+- [Values](../../cep/query-guide/value)
+- [Event Types](../../cep/query-guide/event-types)
+- [Functions](../../cep/query-guide/functions/)
+- [Filters](../../cep/query-guide/filters/)
+- [Aggregate Functions](../../cep/query-guide/aggregate-functions)
+- [HAVING|WHERE](../../cep/query-guide/having-where)
+- [GROUP BY](../../cep/query-guide/group-by)
+- [ORDER BY](../../cep/query-guide/order-by)
+- [LIMIT and OFFSET](../../cep/query-guide/limit-and-offset)
+- [Output Rate Limiting](../../cep/query-guide/output-rate-limiting)
+- [In-line Windows](../../cep/windows/windows-queries), which are windows defined in the query.
+
+## Unsupported Stream QL
+
+The following Stream QL and stream worker elements are not supported. If you need to use them in your data pipeline, then you might need to create a stream worker:
+
+- [Custom Script Functions](../../cep/query-guide/custom-script-functions)
+- [JOIN](../../cep/query-guide/join/)
+- [Partitions](../../cep/query-guide/partition/)
+- [Named Windows](../../cep/windows/)
+- [Named Aggregations](../../cep/aggregations/)
+- [Ad Hoc Queries](../../cep/ad-hoc-queries/)
+- Creating any other elements, including [Tables](../../cep/table/), [Sources](../../cep/source/), and [SINKS](../../cep/sink/).
+
+## Syntax
+
+```sql
+SELECT
```

Comment:
@diana-macrometa - `INSERT INTO Output` must be there before the `SELECT`.

Reply:
I copied this from Notion, I figured it was just showing the editable lines.

Reply:
@diana-macrometa :shipit: after this is covered cc @grainier-macrometa